lib/site_template/about.md

```diff
@@ -6,6 +6,10 @@ permalink: /about/
 
 This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
 
-You can find the source code for the Jekyll new theme at: [github.com/jglovier/jekyll-new](https://github.com/jglovier/jekyll-new)
+You can find the source code for the Jekyll new theme at:
+{% include icon-github.html username="jglovier" %} /
+[jekyll-new](https://github.com/jglovier/jekyll-new)
 
-You can find the source code for Jekyll at [github.com/jekyll/jekyll](https://github.com/jekyll/jekyll)
+You can find the source code for Jekyll at
+{% include icon-github.html username="jekyll" %} /
+[jekyll](https://github.com/jekyll/jekyll)
```

Comment:
It's meant to be an example of in using the icons inline.

It looks like this:
---
<img width="565" alt="2015-10-20 20_07_43-store" src="https://cloud.githubusercontent.com/assets/43209/10624441/58005d6c-7766-11e5-8b80-3b8523f74bb3.png">